pimatic-led-light.coffee

```diff
@@ -17,9 +17,9 @@ module.exports = (env) ->
     init: (app, @framework, @config) =>
       deviceConfigDef = require("./led-light-schema")
 
-      @framework.deviceManager.registerDeviceClass "LedLight",
+      @framework.deviceManager.registerDeviceClass "IWYLed",
```

Comment:
Yes, I aware that this is a blueprint. I just wanted to prof that I got it right.